model-optimizer/extensions/middle/AddMeanScaleValues_test.py

```diff
@@ -44,7 +46,19 @@
 }
 
 
+def set_graph_attrs(graph, graph_ref):
+    for node in graph.get_data_nodes():
+        node['fw_tensor_debug_info'] = ['data_attributes', 0]
+    for graph_ref in graph_ref.get_data_nodes():
+        graph_ref['fw_tensor_debug_info'] = ['data_attributes', 0]
```

Comment:
Using this approach the `'fw_tensor_debug_info'` attribute is set for all data nodes in the graph. But it should be set to the data node of the `Parameter`. The rest data nodes should not be affected.